zebrad/tests/acceptance.rs

```diff
@@ -1699,9 +1699,11 @@ fn lightwalletd_integration() -> Result<()> {
     // zcash/lightwalletd exits with a fatal error here.
     // adityapk00/lightwalletd keeps trying the mempool,
     // but it sometimes skips the "Method not found" log line.
+    // If the `getrawmempool` method is implemented we should have an error
+    // when looking for the provided regex.
     let result =
         lightwalletd.expect_stdout_line_matches("(Mempool refresh error: -32601: Method not found)|(Another refresh in progress, returning)");
-    let (_, zebrad) = zebrad.kill_on_error(result)?;
+    assert!(result.is_err());
```

Comment:
Maybe we can just remove this assertion, from lines 1699 to 1706? I mean, it will likely not fail from now on because the method is implemented :tada:

Reply:
Yea, it seems ok to just to that. https://github.com/ZcashFoundation/zebra/pull/3851/commits/e61d491afb297abc38006a1d3447888cb9cbe77b